Shield-Changelog.md

```diff
@@ -26,6 +26,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Support Role Based Access Admin
 - Support client certificate auth to sites - proxyless
 - New Format for Error Pages 
+- Suspected Sites in Policy  
+- Add user location on URL
```

Comment:
Is it also in on-prem?

Reply:
suspected sites in policy is included on prem
this contain several changes:
1. new security category on policies table (phishing, high risk , medium etc)
2. new security message that can be enabled or disabled
3. there is an option at the admin to select what  phishing / malware vendors  to enable

Add User location on Url:
we have a new field for it at the admin UI